.circleci/config.yml

```diff
@@ -335,6 +335,10 @@ jobs:
       # Pull in all submodules (inc. rust-fil-proofs)
       - run: git submodule update --init
 
+      - run:
+          name: Generate fixtures
+          command: go build -o ./gengen/gengen ./gengen
+
       - run:
```

Comment:
I think this is already happening in the Build step, maybe `fixtures` is not persisted to the workspace ?
from https://circleci.com/gh/filecoin-project/go-filecoin/11562
```
Building gengen utils...
go build -o ./gengen/gengen ./gengen
Building faucet...
go build -o ./tools/faucet/faucet ./tools/faucet/
Building genesis file server...
go build -o ./tools/genesis-file-server/genesis-file-server ./tools/genesis-file-server/
Generating genesis...
./gengen/gengen --keypath fixtures --out-car fixtures/genesis.car --out-json fixtures/gen.json --config ./fixtures/setup.json
```

Reply:
@gmasgras I wasn't sure if the fixtures are sensitive from a security perspective and should be persisted to workspace. I intentionally generated them in the docker job, in case. if there's nothing sensitive, I'll persist to workspace

Reply:
@gmasgras changes made as requested